docs/builders/genai-builder.md

```diff
@@ -0,0 +1,29 @@
+# Observability Builder
+
+The `GenAIBuilder` allows you to get started with a builder class to configure required addons as you prepare a blueprint for setting up Generative AI EKS cluster.
+
+## Usage 
+
+### Generative AI on EKS Cluster
+
+The below usage helps you with a demonstration to use `GenAIBuilder` to setup required addons as you prepare a blueprint for setting up Generative AI on a new EKS cluster.
+
+```typescript
+import { Construct } from 'constructs';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+import { ObservabilityBuilder } from '@aws-quickstart/eks-blueprints';
+
+export default class GenAIShowcase {
```

Comment:
What showcase will customer get after deploying this other a vanilla EKS cluster with some addons?